templates/pages/resource.json

```
@@ -0,0 +1,126 @@
+{
+  "name": "Resource",
```

Existing page names are pluralized, e.g. Executions or Local Blueprints

```suggestion
  "name": "Resources",
```